arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1418,10 +1410,6 @@
 				<string>ChooseCameraController</string>
 				<key>descriptionText</key>
 				<string>Control the behavior of the camera in a scene.</string>
-				<key>dependency</key>
```

Comment:
This dependency is needed for the Choose Camera Controller sample. Please revert this change.

Reply:
I am also confused here because there's no array of strings that follows this line if I were to add it. Can this line of code work without a succeeding array of strings?